package.json

```diff
@@ -36,17 +36,18 @@
     "url": "git://github.com/opentable/spur-common.git"
   },
   "dependencies": {
-    "spur-ioc": "^0.2.1",
-    "spur-errors": "^0.2.1",
-    "spur-config": "^1.0.0",
-    "lodash": "^3.0.0",
     "bluebird": "^2.9.3",
-    "winston": "^0.8.3",
     "superagent": "^3.0.0",
-    "moment-timezone": "^0.3.0",
+    "colors": "1.1.0",
     "form-data": "^0.2.0",
     "joi": "^5.1.0",
-    "colors": "1.1.0"
+    "lodash": "^4.0.0",
+    "moment-timezone": "^0.3.0",
+    "spur-config": "^1.0.0",
+    "spur-errors": "^0.2.1",
+    "spur-ioc": "^0.2.1",
+    "superagent": "^1.7.2",
```

Comment:
Seems you're rolling back the version from `^3.0.0` to `1.7.2`. Can you please restore and update to `^3.3.1`? For the same reason as above.